apps/ledger-live-desktop/src/renderer/families/celo/ActivateFlowModal/steps/StepVote.tsx

```diff
@@ -84,7 +84,13 @@ const StepVote = ({
   const unit = getAccountUnit(account);
   return (
     <Box flow={1}>
-      <TrackPage category="Celo Activate" name="Step 1" />
+      <TrackPage
+        category="Celo Activate"
+        name="Step 1"
```

Comment:
would it not be better for this to be `Step Vote?

Reply:
These are not our attributes. I understand blockchain has followed analytics specifications.